test/mason/pkgconfig-tests/openssl.good

```diff
@@ -0,0 +1,17 @@
+
+[root]
+name = "openssl"
+version = "0.1.0"
+chplVersion = "CHPL_CUR_FULL..CHPL_CUR_FULL"
+
+[external]
+
+[external.pkgconfig]
+
+[external.pkgconfig.openssl]
+name = "openssl"
+version = "0.9.8zh"
```

Comment:
Hard-coding versions like this into the `.good` files may be fragile to system updates. Could we use a prediff to plug in `openssl version | awk '{print $2}'`  (or the equivalent for other packages) into the version field to make these tests more resilient?

Reply:
Actually, `pkg-config --modversion <package>` would be better than my earlier example, but you probably already figured that out.